resolvers/__tests__/UserActions.spec.js

```diff
@@ -7,7 +7,13 @@ const { Mutation, Query } = require('../UserActions')
 const faker = require('faker')
 const _ = require('lodash')
 
-afterAll(() => Model.knex().destroy())
+afterAll(async () => {
+  await UserAction.query().delete()
```

Comment:
Why the delete on every model is needed?

Reply:
I think we need to clean before and after every test, if this test executes at last it will leave records in the test database.